controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -220,6 +290,11 @@ func (r *StorageConsumerReconciler) own(resource metav1.Object) error {
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *StorageConsumerReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	ctx := context.Background()
+	if err := mgr.GetCache().IndexField(ctx, &corev1.Secret{}, util.OwnerUIDIndexName, util.OwnersIndexFieldFunc); err != nil {
+		return fmt.Errorf("unable to set up FieldIndexer for Secret's owner uid: %v", err)
+	}
+
```

Comment:
We are not listing based on the ownerIndex do we need this?

Reply:
done, but didn't change the rename of index in other file as it is good to prefix fields w/ some string, here `index:...`